src/app/space/settings/collaborators/collaborators.component.html

```diff
@@ -47,21 +47,28 @@ <h3 class="collaborators-header">Collaborators of {{context.space.attributes.nam
   </div>
 </div>
 
-<modal #addCollaborators modalClass="chromeless-modal">
-  <modal-content class="chromeless-modal-content">
-    <add-collaborators-dialog [host]="addCollaborators" [spaceId]="context?.space.id" (onAdded)="addCollaboratorsToParent($event)"></add-collaborators-dialog>
-  </modal-content>
-</modal>
-
+<div class="modal fade" bsModal #modalAdd="bs-modal" tabindex="-1" role="dialog" aria-hidden="true" (onShown)="onShowHandler()">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-body">
+          <add-collaborators-dialog #addCollabDialog [host]="modalAdd" [spaceId]="context?.space.id" (onAdded)="addCollaboratorsToParent($event)"></add-collaborators-dialog>
```

Comment:
indentation

Reply:
Running this patch locally and comparing vs prod, this dialog seems to have lost some of the styling for its title.

Before:
![screenshot from 2018-02-08 13-40-02](https://user-images.githubusercontent.com/3787464/35991571-979f4f62-0cd5-11e8-8ce9-9bad8ad7585b.png)

After:
![screenshot from 2018-02-08 13-40-32](https://user-images.githubusercontent.com/3787464/35991593-a8a7ab38-0cd5-11e8-892f-d7fb7ad9a00d.png)

Reply:
This occurs locally ? My local copy doesn't have this issue .. 
<img width="1440" alt="screen shot 2018-02-08 at 3 35 39 pm" src="https://user-images.githubusercontent.com/17230733/35996896-cb5a7bf0-0ce5-11e8-8add-baa34500fa88.png">